docs/2.9.0/docs/query/pqs-user-guide.rst

```diff
@@ -1,466 +1,1741 @@
-.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
-.. SPDX-License-Identifier: Apache-2.0
-
-Participant Query Store User Guide
-##################################
+PQS Technical User Documentation
+================================
 
 Introduction
-************
+------------
+
+PQS (Participant Query Store) is a component that extends the reach of
+the ledger - by making it available to SQL. This enables users to use
+SQL to query the state of the ledger, as well as the history of how that
+state came to be.
+
+Audience
+~~~~~~~~
+
+This documention provides guidance on what PQS achieves, how to use it,
+and it's operation and limitations. The following types of users are
+likely to find PQS useful:
+
+-  Application developers: to query the ledger to display information to
+   users, integrate or carry out automated business processes, and
+   debugging.
+-  Business analysts: to access and analyze ledger data with ad-hoc
+   investigations to understand operational usage.
+-  Support analysts: to diagnose any problems on a live system, when
+   sophisticated inspection of the ledger is required.
+-  Application operators: to take a snapshot of the ledger, either with
+   or without history in order to analyse and investigate.
+-  Report analysts: to provide access to ledger data information for
+   analysis and aggregation with standard reporting tools, on a
+   historical or real-time basis.
+
+Purpose
+~~~~~~~
+
+PQS can be described as a kind of
```

Comment:
One of the concepts that needs to be emphasized is that:  "PQS is part of the PN."  We want customers to understand this.  However, these two paragraphs deemphasize this.  Can we reoritent the content to emphasize this?

I'm also not sure this is true:  "Providing read access to the data via a traditional SQL database decouples the reader from a specific Participant " since a PQS is tied directly to a PN.